helper/reports.go

```diff
@@ -0,0 +1,76 @@
+package helper
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+)
+
+func ProcessDownloadReports(fileID int, alwaysApproved bool, generate string, output string) (bool, error) {
+	var resultID int
+
+	fmt.Println("Warning: This process will download report file to system.")
```

Comment:
Put the warning after checking for flags. There is no points printing a warning if we are not going to act on it 😛

Reply:
@rajan262 As per requirements this warning message will be always print either accept or not.